[16.0][UPD] l10n_es_igic: Se añade nuevo tipo reducido del 5% IGIC

yapci-sdatos opened the pull request:
A partir del 1 de enero de 2024, se aplica un nuevo tipo reducido del 5% IGIC a las entregas e importaciones de bebidas energéticas y bebidas refrescantes y gaseosas con azúcares o edulcorantes añadidos.

Commit:
[ADD] l10n_es_aeat_mod130: Módulo para la declaración del modelo 130 - IRPF pago fraccionado estimación directa

Commit:
Launchpad automatic translations update.

Commit:
[FIX] l10n_es_aeat_mod130: Corrección del archivo exportado para evitar código 0339 en la presentación.

acysos committed:
[FIX] l10n_es_aeat: compatibilidad de la función export_to_boe para el modelo 340

Commit:
[FIX] l10n_es_aeat_mod130: Tilde en archivo que hace fallar runbot.

Commit:
[FIX] l10n_es_aeat_mod130/mod303: import corregido para que valga también para la v8.

rlizana committed:
[MIG] l10n_es_aeat_mo130

* Clases a CapWords
* PEP8
* Icono del modulo

pedrobaeza committed:
[FIX] l10n_es_aeat_mod130: Problema en la búsqueda cuando los periodos no son trimestrales + PEP8 79 cols

Commit:
[IMP] unificar campos en el módulo base + migrar a la nueva api.

Commit:
[ADD] Add l10n_es_location_nuts addon

pedrobaeza committed:
[FIX+IMP] l10n_es_aeat_*: PEP8 + Refinamientos varios

oca-transbot committed:
OCA Transbot updated translations from Transifex

yajo committed:
Give toponyms wizard the right priority.

This will make it be executed before the NUTS import wizard.

See https://github.com/OCA/l10n-spain/pull/150#issuecomment-148313801

antespi committed:
Complete spanish translation

antespi committed:
Use separate and explicit fields for each NUTS level

hugosantosred committed:
[IMP] Añadir reglas de registro multicompañía a informes financieros e informes AEAT

chienandalu committed:
[MIG] l10n_es_location_nuts: Migration to 10.0

etobella committed:
[ADD] l10n_es_facturae_face

Commit:
[MIG] l10n_es_location_nuts: Migration to 11.0

etobella committed:
[MIG] l10n_es_facturae_face: Migration to 11.0

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

Soluntec committed:
[ADD] l10n_es_payment_order_confirming_XXXX: Modulos exportación de fichero confirming para bancos sabadell, popular y bankia. (#393)

* Exportación de fichero confirming para banco sabadell.
* Exportación de ficheros bancarios según el formato Confirming para Banco Popular.
* Exportación de ficheros bancarios según el formato Confirming para Bankia.

[UPD] Update l10n_es_payment_order_confirming_sabadell.pot

pedrobaeza committed:
[FIX] l10n_es_facturae_face: Avoid some problems + deactivate test

They are failing and breaking branch, so this is the best soluction for now

etobella committed:
[FIX] facturae_face: Certificado en formato DER

etobella committed:
[FIX] facturae_face: Missing parent field

etobella committed:
[FIX] l10n_es_facturae_face: missing sudos

Commit:
[MIG] l10n_es_aeat_mod130: Migration to 11.0

Tardo committed:
[MIG] l10n_es_location_nuts: Migration to 12.0

pedrobaeza committed:
[MIG] l10n_es_aeat_mod130: Finish migration

- Remove redundant fields
- Expenses of difficult justification
- Monetary type in fields
- Adjust BOE export format
- Adjust views
- Test BOE export
- Other fine tuning

etobella committed:
[FIX] facturae_face: ejecutar como sudo debido a cambio del mimetype

Commit:
[12.0][MIG] l10n_es_facturae_face

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_facturae_face/

OCA-git-bot committed:
[UPD] README.rst

ernestotejeda committed:
[MIG] l10n_es_aeat_mod130: Migration to 12.0

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_facturae_face/

CarlesAntoli committed:
Translated using Weblate (Spanish)

Currently translated at 51.4% (73 of 142 strings)

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_mod130
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_mod130/es/

acysos committed:
[ADD] l10n_es_aeat: certificates and soap. AEAT Partner check

[ADD] l10n_es_aeat: certificates and soap. AEAT Partner check

[ADD] l10n_es_aeat: certificates and soap. AEAT Partner check

[ADD] l10n_es_aeat: certificates and soap. AEAT Partner check

[ADD] l10n_es_aeat: certificates and soap. AEAT Partner check

acysos committed:
[FIX] l10n_es_aeat_partner_check: soporte para NIF con y sin "ES"

[ADD] l10n_es_aeat_partner_check: add _parse_aeat_vat_info

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (23 of 23 strings)

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_partner_check
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_partner_check/es/

rlizana committed:
[12.0] l10n_es_aeat_*: [FIX] Caracteres no válidos 'NIF Empresa Desarrollo'

etobella committed:
[12.0][IMP] l10n_es_facturae_face: Add cron to update data, removing reference to mandate

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_facturae_face/

javierobcn committed:
[IMP] l10n_es_aeat_mod130: Add chatter

etobella committed:
[FIX] facturae_face: Error en la validación del correo electrónico

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
l10n_es_facturae_face 12.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_facturae_face/

javiermatos committed:
Setting field 06 for Model 130 to editable. Computing initial value for 06 field using withholdings.

brendapaniagua committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (142 of 142 strings)

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_mod130
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_mod130/ca/

etobella committed:
[REF] l10n_es_facturae_face: Black python code

etobella committed:
[MIG] l10n_es_facturae_face

eugenichafer committed:
Added translation using Weblate (Catalan)

oca-travis committed:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
[UPD] README.rst

brendapaniagua committed:
Translated using Weblate (Catalan)

Currently translated at 95.6% (22 of 23 strings)

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_partner_check
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_partner_check/ca/

danimv5 committed:
Added translation using Weblate (Catalan)

danimv5 committed:
Added translation using Weblate (Spanish)

Commit:
[12.0][IMP] l10n_es_aeat_partner_check: Comprobación regargo equiv.

Añade una petición para comprobar si la empresa está sometido al recargo de equivalencia, indicandolo junto con el resto de la información del pedido

Commit:
[12.0][IMP] l10n_es_aeat_partner_check: black, isort, prettier

Commit:
[14.0][MIG] l10n_es_aeat_partner_check: Migration to 14.0

etobella committed:
[IMP] l10n_es_facturae: Add disable_edi_auto configuration

etobella committed:
[FIX] l10n_es_facturae_face: Fix tests

ao-landoo committed:
[FIX] l10n_es_aeat_partner_check: Add missing return values

OCA-git-bot committed:
l10n_es_facturae_face 13.0.1.0.2

Commit:
[IMP] l10n_es_location_nuts: black, isort, prettier

Commit:
[MIG] l10n_es_location_nuts: Migration to 14.0

Commit:
[UPD] Update l10n_es_location_nuts.pot

OCA-git-bot committed:
[UPD] README.rst

josufmcg committed:
[15.0][MIG] l10n_es_aeat_partner_check: Migration to 15.0

Co-authored-by: JuanjoA <jalgaz@gmail.com>
Co-authored-by: josufmcg <josufmc@gmail.com>

ofonseca-pyming committed:
Added translation using Weblate (Catalan)

victoralmau committed:
[IMP] l10n_es_payment_order_confirming_sabadell: black, isort, prettier

victoralmau committed:
[MIG] l10n_es_payment_order_confirming_sabadell: Migration to 14.0
TT33376

[UPD] README.rst

DarioLodeiros committed:
Update aeat_partner_vat string: AEAT VAT

Modificación del string de aeat para evitar la duplicidad del string (p.e. en las vistas search)

etobella committed:
[REF] l10n_es_facturae_face: Move EDI logic here

We will join e.Fact and FACe in a single module

etobella committed:
[MIG] l10n_es_facturae_face: migration to 14.0

Commit:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
[UPD] README.rst

etobella committed:
[FIX] l10n_es_facturae_face: Add lost job

etobella committed:
[IMP] l10n_es_facturae_face: Remove unnecessary parameter

The parameter is no longer necessary

etobella committed:
[IMP] l10n_es_facturae_face: Remove unnecessary listener, passed as template

etobella committed:
[IMP] l10n_es_facturae_face: Use last edi interface

pedroguirao committed:
[IMP] l10n_es_facturae_face: black, isort, prettier

pedroguirao committed:
[MIG] l10n_es_facturae_face: Migration to 15.0

LoisRForgeFlow committed:
[FIX] l10n_es_facturae_face: skip check step.

The backend does not have a check component, so it will raise
a NotImplementedError if we do not skip the check.

Commit:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
[UPD] README.rst

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (34 of 34 strings)

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_aeat_partner_check
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_aeat_partner_check/ca/

jabelchi committed:
Added translation using Weblate (Catalan)

jabelchi committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (51 of 51 strings)

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_facturae_face/ca/

jabelchi committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (40 of 40 strings)

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_facturae_face/es/

Commit:
[15.0][FIX] l10n_es_facturae_face: changes the way the 'inbound_payment_method_line_ids' is entered in the journal in the tests.

OCA-git-bot committed:
l10n_es_facturae_face 15.0.1.0.1

etobella committed:
[FIX] l10n_es_facturae_face: Certificate must be public

etobella committed:
[IMP] l10n_es_facturae_face: Add maintainer

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
l10n_es_facturae_face 15.0.1.0.2

OCA-git-bot committed:
l10n_es_facturae_face 15.0.1.1.0

etobella committed:
[IMP] l10n_es_facturae_face: remove edi_webservice dependancy

etobella committed:
[IMP] l10n_es_facturae_face: Only attach data if data changes

MiquelRForgeFlow committed:
[FIX] l10n_es_facturae: typo in string

LoisRForgeFlow committed:
[FIX] l10n_es_facturae_face: validate fields before creting exchange record.

We check fields now to raise an error to the user, otherwise the
error will be raising silently in the queue job.

OCA-git-bot committed:
l10n_es_facturae_face 15.0.1.1.1

Commit:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
l10n_es_facturae_face 15.0.1.1.2

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_facturae_face/

Commit:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
l10n_es_facturae_face 15.0.1.2.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_facturae_face/

victoralmau committed:
[FIX] l10n_es_facturae_face: Change to the correct field name to fix tests.

etobella committed:
[FIX] l10n_es_facturae_face: Fix changes applied on EDI

yajo committed:
[FIX] l10n_es_*: remove pyOpenSSL and cryptography dependencies

- pyOpenSSL was replaced by cryptography and is no longer used.
- cryptography has a specific version dependency specified in `l10n_es_aeat`, which in turn is dependency of all these modules. So there's no real need to indicate that dependency in these files.

@moduon MT-1067

OCA-git-bot committed:
l10n_es_facturae_face 15.0.1.2.1

etobella committed:
[FIX] l10n_es_facturae_face: Fix cryptography dependancy

xAdrianC-Kernet committed:
[FIX] *: replace tabs for spaces

OCA-git-bot committed:
l10n_es_facturae_face 15.0.1.2.2

pedrobaeza committed:
[IMP] l10n_es_aeat_mod130: pre-commit stuff

pedrobaeza committed:
[MIG] l10n_es_aeat_mod130: Migration to 14.0

- Standard procedure
- Switch to computed fields
- AEAT base adaptation
- Tests adaptation

OCA-git-bot committed:
l10n_es_aeat_partner_check 15.0.1.1.0

Commit:
[IMP] l10n_es_payment_order_confirming_sabadell: Convertir float/int

l10n_es_payment_order_confirming_sabadell 14.0.1.0.1

etobella committed:
[OU] l10n_es_facturae_face: Add missing migration scripts

eantones committed:
[FIX] l10n_es_facturae_face: "Sending method" field is not displayed on partner's main view only in the child view

The "Sending method" field is not displayed in the partner's view, so it is not possible to set it as "FACe" and therefore EDI records are not created when validating invoices

FernandoRomera committed:
[16.0][MIG] l10n_es_aeat_partner_check: Migration to 16.0

Commit:
[UPD] Update l10n_es_aeat_partner_check.pot

OCA-git-bot committed:
[UPD] README.rst

Commit:
[FIX] l10n_es_payment_order_confirming_sabadell: Align left

l10n_es_payment_order_confirming_sabadell 14.0.1.0.2

pedrobaeza committed:
[FIX] l10n_es_aeat_mod130: Negative/To compensate is also for result=0

OCA-git-bot committed:
l10n_es_facturae_face 15.0.1.2.3

yajo committed:
[FIX] l10n_es_aeat_partner_check: permitir a todos escribir en res.partner

Sin esta corrección, los usuarios estaban teniendo este error al escribir en algunos partners:

> No puedes ingresar a los registros 'Certificados AEAT' (l10n.es.aeat.certificate)
>
> Se permite esta operación para los grupos siguientes:
> - Accounting/Advisor
>
> Ponte en contacto con tu administrador para pedirle acceso si es necesario

@moduon MT-2308

Co-authored-by: Loida Vilaplana <104626797+loida-vm@users.noreply.github.com>

Commit:
[FIX] l10n_es_payment_order_sabadell: Missing string cast

l10n_es_payment_order_confirming_sabadell 14.0.1.0.3

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 11.7% (16 of 136 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_aeat_mod130
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_aeat_mod130/sl/

pedrobaeza committed:
[FIX] l10n_es_aeat_mod130: Type for 0 result must be N

jesusramoneda committed:
[IMP] l10n_es_payment_order_confirming_sabadell: pre-commit stuff

jesusramoneda committed:
[MIG] l10n_es_payment_order_confirming_sabadell: Migration to 16.0

etobella committed:
[FIX] l10n_es_facturae_face: Set some record as noupdate False

Commit:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_facturae_face/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (47 of 47 strings)

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_facturae_face/es/

etobella committed:
[FIX] l10n_es_facturae_face: buttons where hidden if not coming from an ancient version

OCA-git-bot committed:
l10n_es_facturae_face 15.0.2.2.0

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

etobella committed:
[FIX] l10n_es_facturae_face: Set rules in the new way

etobella committed:
[FIX] l10n_es_facturae_face: improve readme

carolinafernandez-tecnativa committed:
[MIG] l10n_es_aeat_mod130: Migration to 15.0

etobella committed:
[IMP] l10n_es_facturae_face: Improve readme

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (135 of 135 strings)

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_aeat_mod130
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_aeat_mod130/es/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_location_nuts
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_location_nuts/

etobella committed:
[MIG] l10n_es_facturae_face: Migration to 16.0

Commit:
[UPD] Update l10n_es_facturae_face.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_facturae_face/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_location_nuts
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_location_nuts/es/

pedrobaeza committed:
[FIX] l10n_es_facturae: Allow invoicing users to set Facturae

As it was, only accountant users can set Facturae check, as the field
was inside a group with such restriction. It doesn't matter that the
field itself has another less restricting group.

Thus, we put the initial check in the same group as the rest, and
move the visibility attributes to each field.

TT47097

OCA-git-bot committed:
[BOT] post-merge updates

yajo committed:
[FIX] l10n_es_facturae_face: upgrade zeep and its mocks

In
https://github.com/mvantellingen/python-zeep/commit/decbb761a78c4ea06fecdcfb24d90e248cd3a3dd
zeep moved the `ServiceProxy` class to a different file, and our test
mocks started to fail. This made those tests flaky, making them randomly
fail, as you can see in
https://github.com/OCA/l10n-spain/pull/3394#issuecomment-1908583904 and
some other comments below.

I'm now setting a lower bound in the zeep requirement and mocking the
correct path to avoid those problems in the future.

In any case, there are still more possible connection problems that I
cannot fix at the moment because... well, the server is down and I
cannot obtain the WSDL file. For those cases, we will just record the
failure and skip the test.

I took the chance to DRY all those mocks and upgrade some outdated
import patterns that are not relevant in Odoo 16 anymore.

These tests are now labeled with the correct tags, so they can be
completely skipped for those of us that don't care about external tests.

@moduon MT-4878

OCA-git-bot committed:
[BOT] post-merge updates

etobella committed:
Revert "[FIX] l10n_es_facturae_face: upgrade zeep and its mocks"

This reverts commit 76bea7bad780f0fc5335dd6e30f0a3193674aa2a.

etobella committed:
[FIX] l10n_es_facturae_face: Use Prod for testing

etobella committed:
[FIX] l10n_es_aeat,l10n_es_facturae,l10n_es_facturae_face: pin cryptograhy library version

We want to avoid the installation of newer versions that might be incompatible with odoo

OCA-git-bot committed:
[BOT] post-merge updates

carolinafernandez-tecnativa committed:
[16.0][IMP] l10n_es_facturae: Change facturae to commercial field
TT47656

carolinafernandez-tecnativa committed:
[IMP] l10n_es_facturae: Make it work for physical people

victoralmau committed:
[MIG] l10n_es_payment_order_confirming_sabadell: Migration to 16.0

TT47835

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (26 of 26 strings)

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_payment_order_confirming_sabadell
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_payment_order_confirming_sabadell/es/

OCA-git-bot committed:
[BOT] post-merge updates

jabelchi committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (48 of 48 strings)

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_facturae_face/ca/

jabelchi committed:
Translated using Weblate (Catalan)

Currently translated at 95.6% (22 of 23 strings)

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_payment_order_confirming_sabadell
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_payment_order_confirming_sabadell/ca/

EmilioPascual committed:
[IMP] l10n_es_aeat_mod303: Return last period

Co-Authored-By: Pedro M. Baeza <pedro.baeza@tecnativa.com>

omar7r commented:
Faltarían los mapeos en las posiciones fiscales. Gracias por el aporte

yapci-sdatos commented:
Ya hemos modificado el archivo de mapeos, pero uno de los test nos sigue dando error: 2024-05-08 14:14:25,524 443 ERROR odoo odoo.addons.l10n_es_aeat_sii_oca.tests.test_l10n_es_aeat_sii: ERROR: TestL10nEsAeatSii.test_tax_agencies_production.

nos podríais orientar? gracias por la ayuda.

Commit:
[IMP] l10n_es_aeat: Refactor para futura implementación Veri*FACTU.

Creación de un nuevo modelo en `l10n_es_aeat` llamado *aeat_mixin*, del
que heredan *sii.mixin* (ya existente) y un nuevo *verifactu.mixin*.
Refactor de *sii.mixin* pasando funcionalidad a *aeat_mixin*
Se han renombrado varios campos sii_* a aeat_* ya que serán utilizados
por ambos.

pedrobaeza commented:
Hola @yapci-sdatos haz squash por favor de los dos commits en uno, y el mensaje de commit debe ser con el siguiente formato y en inglés:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#commit-message

Podría valer lo siguiente:

```
[IMP] l10n_es_igic: Add 5% reduced VAT

Since the 1st of January of 2024, there's a new 5% reduced type for deliveries and imports of energy, soft and with added sugars or sweeteners drinks.
```

Si la explicación extendida es difícil de traducir, se puede perdonar que no sea en inglés.

Commit:
[16.0][MIG] l10n_es_location_nuts

@moduon MT-1811

Commit:
[UPD] Update l10n_es_location_nuts.pot

OCA-git-bot committed:
[BOT] post-merge updates

Tisho99 committed:
[IMP] l10n_es_aeat_mod303_vat_prorate: add special vat prorate

giarve committed:
[FIX] l10n_es_payment_order_confirming_sabadell: Fix search domain confirming payment method in bank journal

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat_mod347/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat_mod111
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat_mod111/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat_mod390
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat_mod390/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat_mod349
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat_mod349/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_facturae
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_facturae/

ofonseca-pyming committed:
Translated using Weblate (Catalan)

Currently translated at 99.3% (158 of 159 strings)

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_vat_book
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_vat_book/ca/

carolinafernandez-tecnativa committed:
[IMP] l10n_es_aeat_mod130: pre-commit auto fixes

carolinafernandez-tecnativa committed:
[MIG] l10n_es_aeat_mod130: Migration to 17.0

TT50118

carolinafernandez-tecnativa committed:
[IMP] l10n_es_payment_order_confirming_sabadell: pre-commit auto fixes

carolinafernandez-tecnativa committed:
[MIG] l10n_es_payment_order_confirming_sabadell: Migration to 17.0

TT50083

FernandoRomera committed:
[IMP] l10n_es_location_nuts: pre-commit auto fixes

FernandoRomera committed:
[MIG] l10n_es_location_nuts: Migration to 17.0

OCA-git-bot committed:
Merge PR #3672 into 17.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat.pot

Commit:
[UPD] Update l10n_es_aeat_mod130.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat_mod130
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat_mod130/

Tisho99 committed:
[IMP] l10n_es_vat_prorate: move prorate type to prorate level and add constraint

OCA-git-bot committed:
Merge PR #3673 into 17.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_location_nuts.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_location_nuts
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_location_nuts/

manuelregidor committed:
[IMP] l10n_es_facturae_face: black, isort, prettier

manuelregidor committed:
[MIG]l10n_es_facturae_face: Migration to 17.0

OCA-git-bot committed:
Merge PR #3674 into 17.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_payment_order_confirming_sabadell.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_payment_order_confirming_sabadell
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_payment_order_confirming_sabadell/

OCA-git-bot committed:
Merge PR #3687 into 17.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat_mod303.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat_mod303
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat_mod303/

Tisho99 committed:
[IMP] l10n_es_aeat_mod303_vat_prorate: adaptation to dependences refactor

OCA-git-bot committed:
Merge PR #3678 into 17.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_vat_prorate.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_vat_prorate
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_vat_prorate/

pedrobaeza committed:
[FIX] Remove pandoc

OCA-git-bot committed:
Merge PR #3656 into 17.0

Signed-off-by etobella

Commit:
[UPD] Update l10n_es_aeat_mod303_vat_prorate.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat_mod303_vat_prorate
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat_mod303_vat_prorate/

Tisho99 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (36 of 36 strings)

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_vat_prorate
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_vat_prorate/es/

Tisho99 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (36 of 36 strings)

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_vat_prorate
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_vat_prorate/ca/

Tisho99 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (14 of 14 strings)

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat_mod303_vat_prorate
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat_mod303_vat_prorate/es/

Tisho99 committed:
Added translation using Weblate (Catalan)

Tisho99 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (14 of 14 strings)

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat_mod303_vat_prorate
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat_mod303_vat_prorate/ca/

OCA-git-bot committed:
Merge PR #3710 into 17.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat/

miquelalzanillas committed:
[IMP] l10n_es_aeat_partner_check: pre-commit auto fixes

miquelalzanillas committed:
[MIG] l10n_es_aeat_partner_check: Migration to 17.0

OCA-git-bot committed:
Merge PR #3701 into 17.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat_partner_check.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_aeat_partner_check
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_aeat_partner_check/

pedrobaeza commented:
@yapci-sdatos ¿Puedes completar lo que se ha pedido?

manuelregidor committed:
[FIX]l10n_es_aeat: Wrong field for res.company

OCA-git-bot committed:
Merge PR #3723 into 17.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
Revert "[FIX] l10n_es_aeat_mod349: origin amount incorrect"

This partially reverts commit 0e7847eb70314e42f89e098a306a03417e5f5950,
amended later by 0e93183c9313675a1e37e0e1c3a3faf941abe5fb.

OCA-git-bot committed:
Merge PR #3683 into 17.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_facturae_face.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-17.0/l10n-spain-17.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_facturae_face/

OCA-git-bot committed:
Merge PR #3743 into 17.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_aeat: Take into account inactive tax templates

Since odoo/odoo#179797, IVA 5% taxes templates are deactivated, and
if you calculate an AEAT report containing them, they are not included
as not being included in the m2m field.

This commit takes this situation into account.

TT51059

OCA-git-bot committed:
Merge PR #3746 into 17.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

yapci-sdatos commented:
Ya mismo lo completo, disculpa por la tardanza.
